setup.R

```diff
@@ -1,6 +1,6 @@
 # Install civis R client
 options(unzip='internal');
-devtools::install_github('civisanalytics/civis-r', ref = 'v0.9.1', upgrade_dependencies = FALSE);
+devtools::install_github('civisanalytics/civis-r', ref = 'v1.1.0', upgrade_dependencies = FALSE);
```

Comment:
any reason we have to install via github? Can we install from Cran?

Reply:
Can pin the version if we install from github, but not from CRAN.